src/integrators/integrator_utils.jl

```diff
@@ -466,10 +484,11 @@ function reset_fsal!(integrator)
   end
   # Do not set false here so it can be checked in the algorithm
   # integrator.reeval_fsal = false
+  nothing
 end
 
 function (integrator::ODEIntegrator)(t,deriv::Type=Val{0};idxs=nothing)
   current_interpolant(t,integrator,idxs,deriv)
 end
 
-(integrator::ODEIntegrator)(val::AbstractArray,t::Union{Number,AbstractArray},deriv::Type=Val{0};idxs=nothing) = current_interpolant!(val,t,integrator,idxs,deriv)
+(integrator::ODEIntegrator)(val::AbstractArray,t::Union{Number,AbstractArray},deriv::Type=Val{0};idxs=nothing) = (current_interpolant!(val,t,integrator,idxs,deriv); nothing)
```

Comment:
This change might be breaking. Also, in Base, mutating functions generally return something, e.g.
```julia
julia> reverse!(collect(1:3))
3-element Array{Int64,1}:
 3
 2
 1

julia> push!(ones(2), 1)
3-element Array{Float64,1}:
 1.0
 1.0
 1.0
```

Reply:
something non-deterministically broke a split methods test. I don't think it's this though.

Reply:
Sure, we can drop this nothing